Packs/MitreCaldera/Integrations/MitreCaldera/MitreCaldera.py

```diff
@@ -1323,6 +1325,133 @@ def get_objectives_command(client: Client, args: Dict[str, Any]) -> CommandResul
     return command_results
 
 
+def normalize_date_str(date_str: str) -> str:
```

Comment:
see if you can remove **normalize_date_str** method and use **dateparser.parse** instead:
https://dateparser.readthedocs.io/en/latest/#basic-usage